db/models.js

```diff
@@ -87,4 +87,32 @@ exports.fetchCommentsById = (id) => {
           }
         return comments.rows;
     });
-}
\ No newline at end of file
+}
+
+exports.insertComment = (id, body, author) => {
+  if(isNaN(id)) {
+    return Promise.reject({
+      status: 400,
+      msg: `Incorrect data type`,
+    });
+  } else if(!body || !author) {
+    return Promise.reject({
+      status: 400,
+      msg: `Post body incorrect`,
+    });
+  } 
+return db.query(`SELECT articles.article_id FROM articles`).then((article_id)=>{
+  const idExists = article_id.rows.filter(element => element.article_id == id)
```

Comment:
I recommend using `WHERE article_id = ${id}` and then if it throws a psql error/empty then do `Promise.reject`.